tsl/test/expected/cagg_usage-16.out

```diff
@@ -807,3 +822,31 @@ SELECT hypertable_name, view_name, invalidate_using
 -----------------+-----------+------------------
 (0 rows)
 
+-- Check that trigger-based invalidation works as well, and give
+-- proper errors when trying to modify them.
+CREATE MATERIALIZED VIEW magic3_day_summary_trigger
+  WITH (timescaledb.continuous, timescaledb.invalidate_using = 'trigger')
+    AS SELECT time_bucket('1 day', time), device, avg(value)
+         FROM magic3
+     GROUP BY 1,2;
+NOTICE:  refreshing continuous aggregate "magic3_day_summary_trigger"
+CREATE MATERIALIZED VIEW magic3_week_summary_trigger
+  WITH (timescaledb.continuous, timescaledb.invalidate_using = 'trigger')
+    AS SELECT time_bucket('1 week', time), device, avg(value)
+         FROM magic3
+     GROUP BY 1,2;
+NOTICE:  refreshing continuous aggregate "magic3_week_summary_trigger"
+\set ON_ERROR_STOP 0
+-- This should error out since we already are using WAL-based
+-- collection.
+CREATE MATERIALIZED VIEW magic3_week_summary_wal
+  WITH (timescaledb.continuous, timescaledb.invalidate_using = 'wal')
+    AS SELECT time_bucket('1 week', time), device, avg(value)
+         FROM magic3
+     GROUP BY 1,2;
+ERROR:  cannot use wal-based invalidation collection with hypertable that is using trigger-based invalidation collection
+-- This should error out since we cannot add the trigger to the
```

Comment:
Shouldn't this be:

```suggestion
-- This should error out since we cannot add WAL invalidation to the
```

Reply:
Not correct, but I'll clarify the comment.